mirrord/agent/iptables/src/mesh.rs

```diff
@@ -97,10 +97,9 @@ where
     }
 
     async fn unmount_entrypoint(&self) -> IPTablesResult<()> {
-        self.prerouting.unmount_entrypoint().await?;
-        self.output.unmount_entrypoint().await?;
-
-        Ok(())
+        let first_res = self.prerouting.unmount_entrypoint().await;
+        let second_res = self.output.unmount_entrypoint().await;
+        first_res.and(second_res)
```

Comment:
```suggestion
        let prerouting_unmount = self.prerouting.unmount_entrypoint().await;
        let output_unmount = self.output.unmount_entrypoint().await;
        prerouting_unmount.and(output_unmount)
```

Just because I think it looks nicer than enumerating them